tools/c7n_mailer/c7n_mailer/utils.py

```diff
@@ -124,6 +126,20 @@ def get_date_age(date):
     return (datetime.now(tz=tzutc()) - parser.parse(date)).days
 
 
+def resource_tag(resource, k):
+    for t in resource.get('Tags', []):
+        if t['Key'] == k:
+            return t['Value']
+    return ''
+
+
+def resource_owner(resource, k):
```

Comment:
this isn't any different from the resource_tag implementation, also note this representation is specific to aws, also how are these this different then get_resource_tag_value.

Reply:
It seems as though there's a discrepancy with the readme and the actual code these two functions are reference there, though they aren't actually present. I'm comfortable with removing those references from the `README` and pointing users to  `get_resource_tag_value ` since it looks like this function will satisfy both use cases. https://github.com/capitalone/cloud-custodian/blame/master/tools/c7n_mailer/README.md#L373

Reply:
how about kill the existing implementations in the pr, and add alias to 'resource_tag' for the extant implementation (as a much shorter form), and update the readme.

Reply:
Sounds good to me. I've updated, with just referencing `resource_tag` as an alias.